perun-base/src/test/resources/test-schema.sql

```diff
@@ -1685,3 +1673,120 @@ alter table user_ext_source_attr_values add constraint uesattrval_attr_fk foreig
 
 alter table members_sponsored add constraint memspons_mem_fk foreign key (sponsored_id) references members(id);
 alter table members_sponsored add constraint memspons_usr_fk foreign key (sponsor_id) references users(id);
+
+-- these tables must be created after all the referenced primary keys are defined
```

Comment:
I believe you should be able to create tables and add constraints and references after they are added to the related tables. So keep it at the end of create table section and add constraints at the end of alter table section.